code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -933,6 +933,12 @@
 		M.AdjustLoseBreath(2 SECONDS)
 	return list(0, update_flags)
 
+/datum/reagent/medicine/stimulative_agent/changeling
+	id = "stimulative_cling"
+
+/datum/reagent/medicine/stimulative_agent/changeling/on_mob_add(mob/living/L)
+	REMOVE_TRAIT(L, TRAIT_GOTTAGOFAST, id)
```

Comment:
You shouldn't need to remove the trait here, it looks like it's added by the parent call and by overriding this without a ..() it wouldn't be called anyway.
```suggestion
	return 
```